src/gui/updater/appimageupdater.cpp

```diff
@@ -161,14 +161,15 @@ void AppImageUpdater::versionInfoArrived(const UpdateInfo &info)
         return;
     }
 
-    auto dialog = new AppImageUpdateAvailableDialog(currentVersion, newVersion, ocApp()->gui()->settingsDialog());
+    auto widget = new AppImageUpdateAvailableWidget(currentVersion, newVersion, ocApp()->gui()->settingsDialog());
+    widget->setAttribute(Qt::WA_DeleteOnClose);
```

Comment:
I guess the close event will never be emitted as this is a plain widget.
I guess we should explicitly handle the delete in connects, `accepted`...?

Reply:
That's done in the widget: the signal handler for both buttons calls a `close()` after the signal emission.

Reply:
Please fix